ui/helpers/constants/design-system.ts

```diff
@@ -109,6 +110,7 @@ export enum BorderColor {
 export enum TextColor {
   textDefault = 'text-default',
   textAlternative = 'text-alternative',
+  textAlternativeSoft = 'text-alternative-soft',
```

Comment:
Not aware of this color does `textMuted` not work?

Reply:
It's a new color introduced in [Cross-chain Swaps](https://www.figma.com/design/IuOIRmU3wI0IdJIfol0ESu/Cross-Chain-Swaps?node-id=1697-15289&node-type=instance&m=dev). It's not exactly the same as textMuted

Is it ok to do this for now and update the design package as a fast follow? I just have the color defined in css

Reply:
It might be a disparity between the versions of colors in figma and the extension we text-alternative (soft) is just `text-alternative` I'll ask to remove the brackets to prevent confusion. This PR should resolve any color value discrepancies https://github.com/MetaMask/metamask-extension/pull/29117